OpenTabletDriverPlugins/WindowsInk.cs

```diff
@@ -0,0 +1,299 @@
+            pos.X /= TabletProperties.MaxX;
+            pos.Y /= TabletProperties.MaxY;
+
+            // Scale to tablet dimensions (mm)
+            pos.X *= TabletProperties.Width;
+            pos.Y *= TabletProperties.Height;
+
+            // Adjust area to set origin to 0,0
+            pos -= Input.Position;
+
+            // Rotation
+            if (Input.Rotation != 0f)
+            {
+                var tempCopy = new Point(pos.X, pos.Y);
+                pos.X = (tempCopy.X * _rotationMatrix[0]) + (tempCopy.Y * _rotationMatrix[1]);
+                pos.Y = (tempCopy.X * _rotationMatrix[2]) + (tempCopy.Y * _rotationMatrix[3]);
+            }
+
+            // Move area back
+            pos.X += _halfTabletWidth;
+            pos.Y += _halfTabletHeight;
+
+            // Scale to tablet area (ratio of 1)
+            pos.X /= Input.Width;
+            pos.Y /= Input.Height;
+
+            // Scale to display area
+            pos.X *= Output.Width;
+            pos.Y *= Output.Height;
+
+            // Adjust display offset by center
+            pos.X += Output.Position.X - _halfDisplayWidth;
+            pos.Y += Output.Position.Y - _halfDisplayHeight;
+
+            // Clipping to display bounds
+            if (AreaClipping)
+            {
+                if (pos.X < _minX)
+                    pos.X = _minX;
+                if (pos.X > _maxX)
+                    pos.X = _maxX;
+                if (pos.Y < _minY)
+                    pos.Y = _minY;
+                if (pos.Y > _maxY)
+                    pos.Y = _maxY;
+            }
+
+            // Post Filter
+            foreach (IFilter filter in _postFilters)
+                pos = filter.Filter(pos);
+
+            // Setting report position based on Logical Maximum
+            double offsetX = -(32767.0 / Output.Width);
+            double offsetY = -(32767.0 / Output.Height);
+
+            var pos_x = Math.Round(pos.X / _selectedDisplay.Width * 32767.0 + offsetX);
+            var pos_y = Math.Round(pos.Y / _selectedDisplay.Height * 32767.0 + offsetY);
+
+            // Clipping to logical bounds
+            if (pos_x < 0)
+                pos_x = 0;
+            if (pos_x > 32767.0)
+                pos_x = 32767;
+            if (pos_y < 0)
+                pos_y = 0;
+            if (pos_y > 32767.0)
+                pos_y = 32767;
+
+            // Create virtual report
+            double normpressure = (double)report.Pressure / (double)TabletProperties.MaxPressure;
+            double pressure = Math.Round(normpressure * 8191.0);
+            int tipState = 0;
+            tipState |= 1 << 4;
+            // bit position - function
+            // 0 - press
+            // 1 - barrel
+            // 2 - eraser
+            // 3 - Invert
+            // 4 - range
+            var erasing = false;
+            // we need to send an out of range report when switching to and from eraser mode
+            // this should be implemented in the binding process
```

Comment:
Creating a binding class that inherits IBinding and sets this value would be a good idea. The best way IMO to implement this would be some sort of local IPC or maybe a static property that the class calls `_get()` during runtime.